dpl-config.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What is this file used for? Does it need to be on the root level of the repository?

Reply:
It was mistake. Removed